tests_config/require_standalone.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const vm = require("vm");
+
+const sources = [
+  "standalone.js",
+  "parser-babylon.js",
+  "parser-flow.js",
+  "parser-typescript.js",
+  "parser-postcss.js",
+  "parser-graphql.js",
+  "parser-markdown.js",
+  "parser-vue.js"
+].map(filename =>
+  fs.readFileSync(path.join(process.env.PRETTIER_DIR, filename), "utf-8")
+);
+
+const sandbox = vm.createContext();
+vm.runInContext(sources.join(""), sandbox);
+
+module.exports = {
+  formatWithCursor(input, options) {
```

Comment:
The real standalone build also has `format`, `utils`, etc. We should probably have those here, too.

Reply:
@suchipi but we have no tests for them